src/fontra/views/editor/editor.js

```diff
@@ -1411,9 +1411,41 @@ export class EditorController {
       { title: "Cancel", isCancelButton: true },
       { title: "Add", isDefaultButton: true, result: "ok", disabled: true },
     ]);
-    dialog.setContent(glyphsSearch);
+    dialog.setContent(
+      html.div(
+        {
+          style: `
+          grid-row: 2 / -1;
+          display: flex;
+          flex-direction: column;
+        `,
+        },
+        [
+          glyphsSearch,
+          html.div({ class: "dialog-add-to-all-sources-checkbox" }, [
+            html.input({
+              type: "checkbox",
+              id: "add-to-all-sources",
+            }),
+            html.label(
+              {
+                for: "add-to-all-sources",
+              },
+              ["Add to all sources"]
+            ),
+          ]),
+        ]
+      )
+    );
     setTimeout(() => glyphsSearch.focusSearchField(), 0); // next event loop iteration
 
+    let addToAllSources = false;
+    dialog.dialogContent
```

Comment:
A click handler can be specified directly in the `html.input()` dict above, roughly like this:
```js
            html.input({
              type: "checkbox",
              id: "add-to-all-sources",
              onclick: () => {/* code */},
            }),
```